rclrs/src/node/mod.rs

```diff
@@ -110,4 +110,14 @@ impl Node {
             .push(Arc::downgrade(&subscription) as Weak<dyn SubscriptionBase>);
         Ok(subscription)
     }
+
+    /// Returns the subscriptions that have not been dropped yet.
+    pub(crate) fn live_subscriptions(&self) -> Vec<Arc<dyn SubscriptionBase>> {
+        let live_subs = self
```

Comment:
This can be made more succinct by dropping the `live_subs` variable

Reply:
Yep, I at first considered also pruning dead subscriptions in here, but then decided against it. Done.